### PR TITLE
[17665] Otexa: Add Translators for Footwear Exports

### PR DESCRIPTION
**Pull request by ddhogan**

### CODE REVIEW

File: src/main/java/gov/ita/dataloader/ingest/translators/OtexaExportFootwearCsvTranslator.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      Map<String, Integer> headers = csvParser.getHeaderMap();
+
+      List<String> valueFields = headers.keySet().stream()
+        .filter(header -> header.startsWith("Y"))
+        .collect(Collectors.toList());
+
+      for (CSVRecord csvRecord : csvParser.getRecords()) {
+        String country = csvRecord.get("Country");
+        String ctryNum = csvRecord.get("ctrynum");
+        String catId = csvRecord.get("Category");
```
Comment (asadhzaidi) [inline src/main/java/gov/ita/dataloader/ingest/translators/OtexaExportFootwearCsvTranslator.java:50]: Should this be csvRecord.get("CAT_ID")?

**Review by asadhzaidi — CHANGES_REQUESTED** [review]
I've added a comment for line 50

File: src/main/java/gov/ita/dataloader/ingest/translators/OtexaExportFootwearCsvTranslator.java (hunk 1 above)
Comment (ddhogan) [reply src/main/java/gov/ita/dataloader/ingest/translators/OtexaExportFootwearCsvTranslator.java:50]: In this line, the `"Category"` is referencing a specific column in the CSV file the way Rob has titled it.

**Review by asadhzaidi — APPROVED** [review]
Looks good!

**Pull request merged**